addOns/quickstart/src/main/java/org/zaproxy/zap/extension/quickstart/LearnMorePanel.java

```diff
@@ -0,0 +1,196 @@
+import java.awt.GridBagLayout;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.awt.font.TextAttribute;
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.swing.ImageIcon;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.control.Control;
+import org.parosproxy.paros.view.View;
+import org.zaproxy.zap.extension.help.ExtensionHelp;
+import org.zaproxy.zap.utils.DesktopUtils;
+import org.zaproxy.zap.utils.DisplayUtils;
+import org.zaproxy.zap.view.LayoutHelper;
+
+public class LearnMorePanel extends QuickStartSubPanel {
+    private static final long serialVersionUID = 1L;
+
+    private static final String FAQ_LINK = "https://github.com/zaproxy/zaproxy/wiki/FAQtoplevel";
+    private static final String GETTING_STARTED_LINK = "https://github.com/zaproxy/zaproxy/releases/download/2.7.0/ZAPGettingStartedGuide-2.7.pdf";
+    private static final String USER_GROUP_LINK = "https://groups.google.com/group/zaproxy-users";
+    private static final String USER_GUIDE_LINK = "https://github.com/zaproxy/zap-core-help/wiki";
+
+    private JPanel contentPanel;
+    private JLabel lowerPadding;
+    private int paddingY;
+
+    public LearnMorePanel(ExtensionQuickStart extension, QuickStartPanel qsp) {
+        super(extension, qsp);
+    }
+
+    @Override
+    public String getTitleKey() {
+        return "quickstart.learn.panel.title";
+    }
+
+    @Override
+    public JPanel getDescriptionPanel() {
+        JPanel panel = new JPanel(new GridBagLayout());
+        panel.setBackground(Color.WHITE);
+        panel.add(QuickStartHelper.getWrappedLabel("quickstart.learn.panel.message1"),
+                LayoutHelper.getGBC(0, 0, 2, 1.0D, DisplayUtils.getScaledInsets(5, 5, 5, 5)));
+        panel.add(new JLabel(" "), LayoutHelper.getGBC(0, 2, 2, 1.0D, DisplayUtils.getScaledInsets(5, 5, 5, 5))); // Spacer
+        return panel;
+    }
+
+    private JLabel ulJLabel(String text) {
+        JLabel label = new JLabel(text);
+        Font font = label.getFont();
+        Map<TextAttribute, Object> attributes = new HashMap<>(font.getAttributes());
+        attributes.put(TextAttribute.UNDERLINE, TextAttribute.UNDERLINE_ON);
+        label.setFont(font.deriveFont(attributes));
+        return label;
+    }
+
+    @Override
+    public JPanel getContentPanel() {
+        if (contentPanel == null) {
+            contentPanel = new JPanel(new GridBagLayout());
+            contentPanel.setBackground(Color.WHITE);
+            int formPanelY = 0;
+
+            ExtensionHelp extHelp = Control.getSingleton().getExtensionLoader().getExtension(ExtensionHelp.class);
+            boolean isHelpAvailable = extHelp != null && extHelp.isHelpAvailable();
+
+            File guide = new File(Constant.getZapHome() + File.separator + "lang" + File.separator
+                    + Constant.messages.getString("gettingStarted.file"));

```

Comment:
It should use `containsKey` before to avoid logging an error about missing resource message when the Getting Started with ZAP Guide add-on is not installed.